RecoLocalCalo/HcalRecAlgos/test/HcalSevLvlAnalyzer.cc

```diff
@@ -37,17 +36,21 @@
 // class decleration
 //
 
-class HcalSevLvlAnalyzer : public edm::EDAnalyzer {
+class HcalSevLvlAnalyzer : public edm::one::EDAnalyzer<> {
 public:
   explicit HcalSevLvlAnalyzer(const edm::ParameterSet&);
   ~HcalSevLvlAnalyzer();
 
 private:
-  virtual void beginJob();
-  virtual void analyze(const edm::Event&, const edm::EventSetup&);
-  virtual void endJob();
+  void beginJob() override;
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+  void endJob() override;
 
   // ----------member data ---------------------------
+#ifdef THIS_IS_AN_EVENTSETUP_EXAMPLE
+  const edm::ESGetToken<SetupData, SetupRecord> totkSet_;
+#endif
```

Comment:
Now that the event setup happens with the `getData` method, I believe these lines are not needed anymore

Reply:
Removed